LeagueAPI/LeagueAPI/Models/RiotModels/RankedEntry.swift

```diff
@@ -10,28 +10,24 @@ import Foundation
 
 public class RankedEntry: Decodable {
     
-    public var tier: RankedTier
```

Comment:
I still see tier in [the documentation](https://developer.riotgames.com/apis#league-v4/GET_getLeagueEntriesForSummoner).
Maybe they stoped using it, but in this case make it optional, to keep supporting previous data models.